resources/sugarcrm/emailshandler.cpp

```diff
@@ -132,12 +132,14 @@ void EmailsHandler::getExtraInformation(Akonadi::Item::List &items)
             if (pos == -1) {
                 qCWarning(FATCRM_SUGARCRMRESOURCE_LOG) << "Email not found:" << email_id;
             } else {
-                SugarEmail email = items[pos].payload<SugarEmail>();
+                auto item = items.at(pos);
```

Comment:
> I'm never sure with auto: does this make a copy or does that declare item as a ref, given that at() returns a ref?

It makes a copy, and decays, so a C array decays into a pointer.

  const auto &item =